PrivateHeaders/XCTest/XCUIApplicationProcess.h

```diff
@@ -65,7 +65,11 @@
 - (void)terminate;
 - (void)waitForViewControllerViewDidDisappearWithTimeout:(double)arg1;
 - (void)waitForAutomationSession;
+// Before Xcode16-beta5
 - (void)waitForQuiescenceIncludingAnimationsIdle:(BOOL)arg1;
+// Since Xcode16-beta5
+- (void)waitForQuiescenceIncludingAnimationsIdle:(BOOL)arg1 isPreEvent:(BOOL)arg2;
```

Comment:
It looks like here has new ones:
```
- (void)waitForQuiescenceIncludingAnimationsIdle:(_Bool)idle isPreEvent:(_Bool)event;
- (void)waitForQuiescenceIncludingAnimationsIdle:(_Bool)idle usingActivity:(_Bool)activity isPreEvent:(_Bool)event;
```

not sure the using activity one, but I think lg overall

Reply:
I've tested it locally with simulator and it seems ok. Need more feedback to confirm the other method is called independently as I can only see it being invoked from `- (void)waitForQuiescenceIncludingAnimationsIdle:(_Bool)idle isPreEvent:(_Bool)event;` one